scout/reports.md

```diff
@@ -0,0 +1,97 @@
+---
+description: More details on the Docker Scout Advisory Database and CVE-to-package matching service.
+keywords: scanning, analysis, vulnerabilities, Hub, supply chain, security, report, reports
+title: Analysis reports
+---
+
+{% include scout-early-access.md %}
+
+Docker Scout reports help you share a report of the analysis of images in a namespace with your team. The reports helps team members in roles such as security, compliance, and operations to know what vulnerabilities and issues they need to focus on.
+
+## Overview
+
+![A screenshot of the Docker Scout vulnerabilities overview](./images/reports-overview.png)
+The overview tab shows the number of critical, high, medium, and low vulnerabilities over time.
```

Comment:
We should mention the total vulnerabilities is calculated by adding together all the vulnerabilities across all repositories but only taking the most recent image for each repository.

Something like
> The overview tab shows the total number of vulnerabilities across all your scout-enabled repositories, over time. This calculation takes the most recent image in each repository to avoid including old irrelevant images.